horreum-backend/pom.xml

```diff
@@ -40,6 +40,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.regex</groupId>
+            <artifactId>regex</artifactId>
+            <version>${graalvm.version}</version>
+        </dependency>
```

Comment:
```suggestion
        <dependency>
            <groupId>org.graalvm.regex</groupId>
            <artifactId>regex</artifactId>
            <version>${graalvm.version}</version>
            <exclusions>
                <exclusion>
                    <groupId>org.graalvm.sdk</groupId>
                    <artifactId>graal-sdk</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```

Reply:
I think this issue was due to the conflicting versions in quarkus and the declared graal version. I think it is not currently needed for the current version of quarkus and graalvm. Please let me know if you still think it is necessary